CHANGELOG.md

```diff
@@ -7,18 +7,25 @@
 > * [CHANGELOG_1.1x.md](./documentation/changelog/CHANGELOG_1.1x.md) - v1.10.0 to v1.19.0
 > * [CHANGELOG_1.2x.md](./documentation/changelog/CHANGELOG_1.2x.md) - v1.20.0 to v1.29.2
 
-# UNRELEASED
+# Node v1.33.0-rc1 / 2025-05-01
+The Lotus v1.33.0-rc1 release introduces experimental v2 APIs with F3 awareness, featuring a new TipSet selection mechanism that significantly enhances how applications interact with the Filecoin blockchain. This release candidate also adds F3-aware Ethereum APIs via the /v2 endpoint while improving existing /v1 APIs and implements intelligent fallback mechanisms between F3 and Expected Consensus. These /v2 APIs are also exposed through the Lotus Gateway
```

Comment:
```suggestion
The Lotus v1.33.0-rc1 release introduces experimental v2 APIs with F3 awareness, featuring a new TipSet selection mechanism that significantly enhances how applications interact with the Filecoin blockchain. This release candidate also adds F3-aware Ethereum APIs via the /v2 endpoint.  All of the /v2 APIs implement intelligent fallback mechanisms between F3 and Expected Consensus and are exposed through the Lotus Gateway.
```

Removed discussion about /v1 since the changes there are miner.  I don't want someone to think we brought F3 awareness to /v1.

Reply:
Committing this before I make changes on top.